client/js/actions/qbank/assessments.js

```diff
@@ -84,6 +85,7 @@ export function createAssessmentOffered(bankId, assessmentId) {
   return {
     bankId,
     assessmentId,
+    body    : { id: '123' }, // have to send up garbage data to make it work.
```

Comment:
no nothing happens with this ID, it is garbage. Just needs any value sent up in the body with a value for it to work. I could send name: "billy". just needs something